build_psa_compliance.py

```diff
@@ -159,8 +131,6 @@ def _run_cmake_build(cmake_build_dir, args):
                 "-DPSA_INCLUDE_PATHS=" + manifest_include + ";" + suite_include
             )
         else:
-            # Applicable for INITIAL_ATTESTATION, INTERNAL_TRUSTED_STORAGE
```

Comment:
Do we want to keep this comment, and also append to this comment: "CRYPTO"?

Reply:
I think it's not required as its applicable for all suites except `IPC`, which is pretty obvious as `if args.suite == "IPC":` is checked before.

Reply:
OK. Less to maintain